pkg/provenance/provenance.go

```diff
@@ -92,11 +92,14 @@ func CollectProvenance() {
 	fmt.Println("Inside CollectProvenance")
 	readKindCompositionFile()
 	if onMinikube() {
-		parse() //using a sample audit log, because
+		print("Running on Minikube!")
+		filePath := "/tmp/minikube-sample-audit.log"
```

Comment:
Good you found this issue.

I was running into this issue sometime back when I was trying. The path had to be be modified for minikube.